scripts/1.service.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
make `README.md` in `scripts` can show an example of how the scripts can be used?

Reply:
Sure

Reply:
done

Reply:
added